src/librustc/ty/maps.rs

```diff
@@ -547,7 +553,8 @@ define_maps! { <'tcx>
     pub def_symbol_name: SymbolName(DefId) -> ty::SymbolName,
     pub symbol_name: symbol_name_dep_node(ty::Instance<'tcx>) -> ty::SymbolName,
 
-    pub describe_def: meta_data_node(DefId) -> Option<Def>
+    pub describe_def: meta_data_node(DefId) -> Option<Def>,
+    pub def_span: meta_data_node(DefId) -> Span
```

Comment:
This was actually added as part of my last PR. I will remove it form both instances and replace it with just `MetaData`.